book/src/out-of-order-blocks.md

```diff
@@ -39,7 +39,7 @@ proposes a block.
 To make it easy to verify, the vote with the golden ticket must be present in
```

Comment:
we could as well put something in the blob, right?

Reply:
today, entries don't encode leader, only the data plane does

Reply:
Needs to be something that ties the stake and the blockhash together.  So a vote seems like the perfect thing.